starknet_devnet/server.py

```diff
@@ -123,22 +92,17 @@ def load_config(self):
     def load(self):
         return self.application
 
-
 def main():
     """Runs the server."""
 
-    args = parse_args()
-
     # Uncomment this once fork support is added
     # origin = Origin(args.fork) if args.fork else NullOrigin()
     # starknet_wrapper.origin = origin
 
+    args = parse_args(sys.argv[1:])
+    state.set_starknet_wrapper(StarknetWrapper(DevnetConfig(args)))
     load_dumped(args)
```

Comment:
what do you think about?
state.dumper.load_dumped(args)
state.dumper.set_dump_options(args)

It implicates move dumper functions from server to dump.py

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/219/commits/95372005781725c71157cd5ac74ce9e644e9542b